packages/peregrine/src/Router/MagentoRouteHandler.js

```diff
@@ -11,7 +11,7 @@ const mountedInstances = new WeakSet();
 export default class MagentoRouteHandler extends Component {
     static propTypes = {
         apiBase: string.isRequired,
-        children: func,
+        renderRoutingError: func,
```

Comment:
This change shouldn't be necessary.

Reply:
In my opinion, it is a more descriptive name for the parameter. For me it was unclear why when we have an errorState we render children.

Reply:
It's called [function as a child (FAAC)](https://reactjs.org/docs/render-props.html#using-props-other-than-render), and it's a common pattern.

Reply:
yeah, I know, but why do we pass renderError function as a child, usually when I pass function as a child, I expect that I will choose what to render inside this function. so I expect that this function will be called in every case, not only in case of error.